prow/cmd/sinker/main.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	period        = time.Hour
-	maxProwJobAge = 7 * 24 * time.Hour
+	maxProwJobAge = 2 * 24 * time.Hour
```

Comment:
maybe we should make it configurable so we don't need to build new images for changes like this?